DataFormats/EgammaReco/src/classes_def.xml

```diff
@@ -156,8 +156,21 @@
   <class name="edm::Wrapper<edm::RefVector<edm::AssociationMap<edm::OneToOne<vector<reco::CaloCluster>,vector<reco::ClusterShape>,unsigned int> >,edm::helpers::KeyVal<edm::Ref<vector<reco::CaloCluster>,reco::CaloCluster,edm::refhelper::FindUsingAdvance<vector<reco::CaloCluster>,reco::CaloCluster> >,edm::Ref<vector<reco::ClusterShape>,reco::ClusterShape,edm::refhelper::FindUsingAdvance<vector<reco::ClusterShape>,reco::ClusterShape> > >,edm::AssociationMap<edm::OneToOne<vector<reco::CaloCluster>,vector<reco::ClusterShape>,unsigned int> >::Find> >" />
   <class name="edm::Wrapper<edm::RefVector<edm::AssociationMap<edm::OneToOne<vector<reco::SuperCluster>,vector<reco::HFEMClusterShape>,unsigned int> >,edm::helpers::KeyVal<edm::Ref<vector<reco::SuperCluster>,reco::SuperCluster,edm::refhelper::FindUsingAdvance<vector<reco::SuperCluster>,reco::SuperCluster> >,edm::Ref<vector<reco::HFEMClusterShape>,reco::HFEMClusterShape,edm::refhelper::FindUsingAdvance<vector<reco::HFEMClusterShape>,reco::HFEMClusterShape> > >,edm::AssociationMap<edm::OneToOne<vector<reco::SuperCluster>,vector<reco::HFEMClusterShape>,unsigned int> >::Find> >" />
 
-   <class name="std::vector<reco::SuperClusterRef>" />
-   <class name="edm::ValueMap<reco::SuperClusterRef>" />
-   <class name="edm::Wrapper<edm::ValueMap<reco::SuperClusterRef> >" />
+  <class name="std::vector<reco::SuperClusterRef>" />
+  <class name="edm::ValueMap<reco::SuperClusterRef>" />
+  <class name="edm::Wrapper<edm::ValueMap<reco::SuperClusterRef> >" />
+  
+  <class name="reco::EgTrigSumObj" ClassVersion="10">
+   <version ClassVersion="10" checksum="2636406548"/>
```

Comment:
Use 3 as the initial version number of a class which has never been stored before

Reply:
ah thank you I did not know that, much appreicated!